src/components/buttons/BackButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@equinor/eds-core-react';
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import removeSubdirectories from '../../utils/removeSubdirectories';
+import EdsIcon from '../icons/EdsIcon';
+
+type BackButtonProps = {
+    to: string;
+};
+
+const BackButton = ({ to }: BackButtonProps): JSX.Element => {
+    const history = useHistory();
+    const { pathname } = useLocation();
+    return (
+        <Button
+            variant="ghost"
+            onClick={(): void => {
+                //Sends the user one subdirectory back unless otherwise specified in the "to" prop.
```

Comment:
Må si jeg ikke er fan av comentarer. mulig "to" bør bli noe mer beskrivende "sendTo" eller "customURL" osv.